CSS/common_css.css

```diff
@@ -56,33 +56,22 @@ Media query:
     -> the width of the div is redefined as well
  */
 
-@media(max-width: 1070px) {
+@media(max-width: 505px) {
     h1 {
-        font-size: 25px;
+        width: 80%;
     }
 }
-
-@media(max-width: 850px) {
+@media(max-width: 600px) {
```

Comment:
a `min-width` constraint is required here.
if there is no `min-width` constraint this block will be red by all peripherals including the smallest. This means that the smallest will have the most work

Reply:
Changed